examples/set-namespace-ns-unset/README.md

```diff
@@ -0,0 +1,47 @@
+# set-namespace: ns Unset Example
```

Comment:
It seems this is more testing an edge case than a useful example to users. If that's the case, we can move it to `tests/namespace/<test-name>`.